deploying-an-existing-application-using-okteto-ui.md

```diff
@@ -0,0 +1,175 @@
+# Deploying An Existing Application Using Okteto UI
+
+In the previous blog post, you learned how to build a FastAPI CRUD application and deployed it to Okteto using the Okteto CLI tool. In this tutorial, you will be learning how to deploy the application directly from your Okteto dashboard.
+
+## Initial Setup
+
+Start by creating a fork of the application from [GitHub](https://github.com/okteto/fastapi-crud) and then clone it locally. In your local clone, set it up using the following commands:
+
+```bash
+$ cd fastapi-crud
+$ python3 -m venv venv && source venv/bin/activate
+(venv)$ pip install -r requirements.txt
+```
+
+Verify that the setup is complete by running:
+
+```bash
+python3 main.py
+```
+
+## Initial Deployment
+
+In the previous post, you defined an Okteto stack file to facilitate the deployment of your application from the CLI tool. In this section, you will be deploying the application from the dashboard.
+
+However, before deployment, you need to reassign the repository where Okteto will store your application images.
+
+Your current `okteto-stack.yaml` file is similar to the code below:
+
+
+```yaml
+name: fastapi-crud
+services:
+  fastapi:
+    public: true
+    image: DOCKERHUBUSERNAME/fastapi-crud:latest
```

Comment:
It's `DOCKERHUBUSERNAME` because there's an instruction in the previous blogpost to replace it to the reader's Docker hub username. 

I'll wait to see what @rberrelleza decides on replacing `DOCKERHUBUSERNAME` with `DOCKERHUB-USERNAME`.

Reply:
Ok, keeping it seems better knowing it was used last time. I still think we should replace youngestdev with DOCKERHUBUSERNAME and I will make a PR for that now.

Reply:
That makes sense then 👍. How about replacing it with `okteto.dev` instead?

Reply:
agree with @jmacelroy . I think we should leave DOCKERHUBUSERNAME in the main branch, and put okteto.dev in the branch that we are creating for the post.

Reply:
Great then. 👍🏾